pype/plugins/global/publish/submit_publish_job.py

```diff
@@ -366,6 +366,7 @@ def _create_instances_for_aov(self, instance_data, exp_files):
 
             new_instance = copy(instance_data)
             new_instance["subset"] = subset_name
+            new_instance["subsetGroup"] = aov
```

Comment:
we should be grouping by layer, rather than AOV, that would always be a group of one ;)